docs/data/material/getting-started/templates/dashboard/components/MainGrid.js

```diff
@@ -15,65 +18,76 @@ const data = [
     value: '14k',
     interval: 'Last 30 days',
     trend: 'up',
-    data: [2, 4, 3, 5, 7, 6, 8, 10],
+    data: [
+      200, 24, 220, 260, 240, 380, 100, 240, 280, 240, 300, 340, 320, 360, 340, 380,
+      360, 400, 380, 420, 400, 640, 340, 460, 440, 480, 460, 600, 880, 920,
+    ],
   },
   {
     title: 'Conversions',
     value: '325',
     interval: 'Last 30 days',
     trend: 'down',
-    data: [10, 7, 6, 8, 5, 4, 4, 2],
+    data: [
+      1640, 1250, 970, 1130, 1050, 900, 720, 1080, 900, 450, 920, 820, 840, 600, 820,
+      780, 800, 760, 380, 740, 660, 620, 840, 500, 520, 480, 400, 360, 300, 220,
+    ],
   },
   {
     title: 'Event count',
     value: '200k',
     interval: 'Last 30 days',
     trend: 'neutral',
-    data: [5, 4, 6, 3, 4, 3, 7, 6],
-  },
-  {
-    title: 'Engagements',
-    value: '30k',
-    interval: 'Last 30 days',
-    trend: 'up',
-    data: [2, 4, 3, 4, 5, 4, 7, 8],
+    data: [
+      500, 400, 510, 530, 520, 600, 530, 520, 510, 730, 520, 510, 530, 620, 510, 530,
+      520, 410, 530, 520, 610, 530, 520, 610, 530, 420, 510, 430, 520, 510,
+    ],
   },
 ];
 
 export default function MainGrid() {
   return (
-    <React.Fragment>
+    <Box sx={{ width: '100%', maxWidth: { sm: '100%', md: '1700px' } }}>
       {/* cards */}
-      <Grid container spacing={2} columns={12}>
+      <Typography component="h2" variant="h6" sx={{ mb: 2 }}>
+        Overview
+      </Typography>
+      <Grid
+        container
+        spacing={2}
+        columns={12}
+        sx={{ mb: (theme) => theme.spacing(2) }}
+      >
         {data.map((card, index) => (
-          <Grid key={index} xs={6} sm={3} md={2} lg={2.25}>
+          <Grid key={index} xs={12} sm={6} lg={3}>
```

Comment:
Beware that the Grid API will change soon: https://github.com/mui/material-ui/pull/42742. If that PR is merged first, we'll have to solve some merge conflicts. Let me know if you need help in that case.